lib/ferrum_pdf/railtie.rb

```diff
@@ -1,5 +1,11 @@
 module FerrumPdf
   class Railtie < ::Rails::Railtie
+    initializer "ferrum_pdf.assets_helper" do
+      ActiveSupport.on_load(:action_view) do
+        include FerrumPdf::AssetsHelper if FerrumPdf.include_assets_helper_module
```

Comment:
Is there some reason why you'd want to disable this helper?

Reply:
I put it like this to imitate the behavior of the controller helpers, but I can remove it.